tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -78,6 +78,138 @@ def environment_length_bytes(e: dict) -> int:
 
 
 class TestLambdaFunction:
```

Comment:
nit: we could consider grouping these logging config tests under a `TestLoggingConfig` class. 

I usually try ordering the test cases in increasing complexity. Example with some (hierarchical) renaming suggestions:
* TestLoggingConfig.test_advanced_logging_configuration => Cover the basic case first
* TestLoggingConfig.test_log_format_switch
* TestLoggingConfig.test_partial_logging_config_update

Reply:
@Morijarti Do you think the lengthy `advanced_logging_configuration` prefix is necessary now that we have everything grouped in a class?

I think this name in particular is quite long and a bit confusing: `test_function_partial_advanced_logging_configuration_update`
* it's quite long to understand what belongs together: partial config vs. partial update?
* `_function_` doesn't add much (should be clear within Lambda context)
* `_advanced_` doesn't add much (just a confusing name)
* `partial_advanced_logging_configuration` => the name is so long that the `partial_config` part kinda gets lost